tests/python/contrib/test_ethosn/test_convert_equivalents.py

```diff
@@ -227,7 +227,7 @@ def expected():
 @requires_ethosn
 @pytest.mark.parametrize(
     "dtype,shape,constant_shape",
-    [("int16", (1, 16, 12, 4), None)],
+    [("float32", (1, 16, 12, 4), None)],
```

Comment:
Just curious: why the dtype change?

Reply:
The use of `int16` here was just to create an invalid test case, that raises an exception. With `int16` now supported, then using `float32` will make that effect.

Reply:
Got it. Thanks!